js/viz/themes.js

```diff
@@ -140,7 +140,6 @@ function patchTheme(theme) {
     mergeScalar(theme.treeMap.group.border, "color", null, theme.axisColor);
     mergeScalar(theme.treeMap.tile.selectionStyle.border, "color", null, theme.primaryTitleColor);
     mergeScalar(theme.treeMap.group.selectionStyle.border, "color", null, theme.primaryTitleColor);
-    mergeScalar(theme.treeMap.group.label.font, "color", null, theme.secondaryTitleColor);
```

Comment:
Why do we need to remove this assertion? Can we fix it without removing?

Reply:
@pavelgruba
LGTY?

Reply:
We used to get the color to the TreeMap group label from the "secondaryTitleColor" constant. But in some new themes, these colors are different. Now you can manually set this color describing a theme.

Reply:
Ok. Thank you for clarification.